docs/family.md

```diff
@@ -50,6 +50,7 @@ flowchart TD
     bcftools_stats["bcftools stats\n(small variants)"]
     sv_stats["SV stats"]
     cpg_pileup["5mCpG pileup"]
+    methbat["Methbat"]
```

Comment:
minor: "MethBat" (capital "B") is the preferred tool descriptor.  Just going to note it here, but I see it in a couple other places